### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by thiagosaife**
This PR tests the most updated code from develop branch with storyfront's code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,16 +30,6 @@ yarn test:unit
...
-## Usage
-To learn how to use the Storyblok's Design System in your project go to:
-[https://www.storyblok.com/docs/guide/in-depth/design-system](https://www.storyblok.com/docs/guide/in-depth/design-system)
-
-## Thanks
-
-<a href="https://www.chromatic.com/"><img src="https://user-images.githubusercontent.com/321738/84662277-e3db4f80-af1b-11ea-88f5-91d67a5e59f6.png" width="153" height="30" alt="Chromatic" /></a>
-
-Thanks to [Chromatic](https://www.chromatic.com/) for providing the visual testing platform that helps us review UI changes and catch visual regressions.
-
```
Comment (fernandacanto) [inline README.md:39]: I believe this part is important and should not remove the README.

File: src/components/Button/__tests__/Button.spec.js [hunk 2]
```diff
@@ -157,6 +157,7 @@ describe('Test SbButton Component', () => {
 
     it('should have a tooltip with correct text', async () => {
       await wrapper.find('button').trigger('focus')
+      // expect(wrapper.find('.sb-tooltip').text()).toBe(iconDescription)
```
Comment (fernandacanto) [inline src/components/Button/__tests__/Button.spec.js:160]: Is this comment important?

File: src/components/Slideover/Slideover.stories.js [hunk 3]
```diff
@@ -62,18 +62,18 @@ export default {
...
+    },
+    argTypes: {
+      orientation: {
+        name: 'orientation',
+        description:
+          'The prop `orientation` is used to define the position where the `Slideover` component will appear, the options are on the `left` or `right`, the default is on the `right`.',
+        control: {
+          type: 'select',
+          options: ['left', 'right'],
+        },
```
Comment (fernandacanto) [inline src/components/Slideover/Slideover.stories.js:76]: I believe it's changed causes the Chromatic error.

File: src/directives/tooltip/__tests__/VTooltip.spec.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import { mount } from '@vue/test-utils'
 
+// import { mountAttachingComponent } from '../../../utils/tests-utils'
```
Comment (fernandacanto) [inline src/directives/tooltip/__tests__/VTooltip.spec.js:3]: Is this comment important?

**Review by fernandacanto — CHANGES_REQUESTED** [review]
I think the merge is great, we just need to check these little points I commented on. 😉

**Comment by thiagosaife** [comment]
@fernandacanto Thanks for pointing those issues. I fixed them on another Pull Request to develop branch. Also fixed the Chromatic errors.

**Pull request closed without merging**